recipes/libgettext/all/conanfile.py

```diff
@@ -120,7 +120,7 @@ def generate(self):
             if target is not None:
                 tc.configure_args += [f"--host={target}", f"--build={target}"]
 
-            if self.settings.build_type == "Debug" and Version(self.settings.compiler.version) >= "12":
+            if (self.settings.build_type == "RelWithDebInfo" or self.settings.build_type == "Debug") and Version(self.settings.compiler.version) >= "12":
```

Comment:
```suggestion
            if (self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) >= "12") or \
               (self.settings.compiler == "msvc" and Version(self.settings.compiler.version) >= "180"):
```

Not need to check build_type